sources/Makefile

```diff
@@ -82,8 +82,7 @@ postgres:
 # Local API only
 api-only:
 	@echo "+\n++ Make: Building and running API only"
-	@docker-compose -f docker-compose.yml up -d postgres 
-	sleep 5
```

Comment:
Any reason we put this and now removed it?

Reply:
In that particular I’m spinning up DB container and API container separately.
Initially I though a few send sleep will provide enough time to bootstrap db before API container try to connect it.
But in single terminal env that has no effect.
So, I remove it.